config/settings.yml

```diff
@@ -979,6 +979,7 @@
     :keep_reports: 6.months
     :purge_window_size: 100
   :queue_timeout: 1.hour
+  :use_sql_view: false
```

Comment:
Just so I am clear and not interpreting this incorrectly:

What you are doing here is making this opt-in to enable this everywhere, but not something that is shipped to customers out of the box and it is only a tool for potentially fixing problems (granted, one that still probably needs heavy vetting)

Reply:
correct - NOT shipped out of the box.

I asked @dmetzger57 if we should ship this enabled out of the box